_episodes/05-loop.md

```diff
@@ -465,7 +473,7 @@ cp unicorn.dat original-unicorn.dat
 Since the `cp` command does not normally produce any output, it's hard to check 
 that the loop is doing the correct thing. By prefixing the command with `echo` 
```

Comment:
This is not directly related to the goal of this PR, so feel free to leave it alone, but I'm not a fan of how the echo stuff is worded. It makes it seem like "prefixing a command with `echo`" is some piece of magic (rather than something that can be understood in terms of the semantics of `echo` that were taught in episode 4). I would rather word it along the lines of:

> Recall the `echo` command we learned earlier, which prints its arguments to stdout. If we prefix our command with `echo`, everything that follows (including "cp") will be treated as a string and printed to stdout. Our loop will print each command as it *would* have been executed (without actually executing it). The following diagram shows what happens when the modified loop is executed, and demonstrates how the judicious use of `echo` is a good debugging technique.

Maybe that's overly verbose though. (I also question whether the diagram helps or hinders understanding, but I'll refrain from going further into the weeds)